server/routes/bookAVisit.ts

```diff
@@ -467,6 +471,22 @@ export default function routes(
       })
     }
 
+    if (config.apis.notifications.enabled) {
+      try {
+        await notificationsService.sendSms({
+          phoneNumber: visitSessionData.mainContact.phoneNumber,
```

Comment:
Just wondering if it's worth cleaning up the phone number in case spaces in it are an issue...?